nixos/modules/services/desktops/espanso.nix

```diff
@@ -6,19 +6,28 @@ in {
   meta = { maintainers = with lib.maintainers; [ numkem ]; };
 
   options = {
-    services.espanso = { enable = options.mkEnableOption (lib.mdDoc "Espanso"); };
+    services.espanso = {
+      enable = mkEnableOption (lib.mdDoc "Espanso");
+      package = mkPackageOption pkgs "espanso" { };
+      wayland = mkOption {
+        type = types.bool;
+        default = false;
+        description = lib.mdDoc "Use the Wayland build of Espanso.";
+      };
```

Comment:
It would probably cleaner to add the wayland variant to the related options of the package option

Reply:
So create a `services.espanso.package.wayland` option?

Reply:
> It would probably cleaner to add the wayland variant to the related options of the package option

@SuperSandro2000 could you explain exactly what you meant here?

Reply:
Is there a reason to use a second variable (`cfg.wayland`) as opposed to having users specify `package = pkgs.espanso-wayland`? It seems like the latter is more common and a little less complex.